ktor-server/ktor-server-test-host/jvmAndNix/src/io/ktor/server/testing/BaseTest.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.testing
+
+import kotlinx.coroutines.*
+import kotlin.time.*
+
+// TODO: can be extracted from `server-testing` module, fully commonized(and js and mingw targets) and used in all ktor-tests by default
```

Comment:
could you log issues for TODO? and drop todo

Reply:
https://youtrack.jetbrains.com/issue/KTOR-4038 done